CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 CHANGELOG
 =========
+# 4.28.0 / 2024-09-24
+* [IMPROVEMENT] Add ansible managed comment to checks.yaml [#602]
```

Comment:
I think you need to add :
[#602]: https://github.com/DataDog/ansible-datadog/issues/602

Reply:
indeed, thanks for noticing! I added the missing links for earlier releases as well